web/src/components/footer/subcomponents/policies/policies.styles.js

```diff
@@ -0,0 +1,20 @@
+import theme from "common/theme";
+
+const policiesStyles = {
+  container: {
+    color: theme.palette.text.textLight,
+    marginTop: "8px",
+  },
+  linkContainer: {
+    marginTop: "8px",
+  },
+  link: {
+    color: theme.palette.text.subtitleLight,
```

Comment:
Rather than re-using subtitleLight and textLight (cuz it's not a subtitle), can we just make new ones for linkLight and linkLightHovered ? This way it's clear what they are and why we are using them

Reply:
Uh I tried but `theme` has a type and I'm not sure how to add to it

Reply:
You should just be able to edit theme.js and add two more properties on the `text` field

Reply:
bro i tried

Reply:
well guess im just stupid